### DISCUSSION TITLE
希望增加支持微软云openai api功能呢？

### DISCUSSION THREAD

**Discussion by CarsonLam** [chatchat-space/Langchain-Chatchat#1394] [General]
希望增加支持微软云openai api功能呢？

**Comment by imClumsyPanda** [comment]
有计划接入 现阶段也可参考https://github.com/chatchat-space/Langchain-Chatchat/pull/1345

**Comment by zRzRzRzRzRzRzR** [comment]
已经支持

**Reply by Huyueeer** [reply]
我比较好奇这部分的azure-api是怎么支持到fastchat的？worker_address的这个worker是需要遵循什么格式么？
